src/odemis/driver/lakeshore.py

```diff
@@ -69,33 +74,23 @@ class Lakeshore(model.HwComponent):
     def __init__(self, name, role, port, sensor_input='b', output_channel=2, **kwargs):
         """
         A driver for the Lakeshore 310 temperature controller.
-
         name: (str)
         role: (str)
         port: (str) port name. Can be a pattern, in which case all the ports
           fitting the pattern will be tried.
           Use /dev/fake for a simulator
-        sensor_input (str): The sensor input to use, typically 'A' or 'B'
-        output_channel: (int): The channel output to control, typically 1 or 2
+        sensor_input (str): The sensor input to use, typically 'A' or 'B' for model 335 and
+            'A', 'B', 'C' or 'D' for model 350
```

Comment:
It now has two ways to specify it. Please describe both ways. I'd write something like:
`sensor_input (str or dict str->str): either a single input name, or the name of the VigilantAttribute (to be followed by temperature) -> input name`

Same thing for the output_channel.